contents/testing/security-testing.md

```diff
@@ -0,0 +1,91 @@
+<frontmatter>
+  title: Introduction to Security Testing
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to Security Testing
+
+**Author(s): [Ahmed Bahajjaj](https://github.com/madanalogy)**<br>
+Reviewers: [Pending](https://github.com/teammates)
+
+<box id="article-toc">
+
+* [Introduction to Security Testing](#introduction-to-security-testing)
+    * [What Is Security Testing?‎](#what-is-security-testing)
+    * [Why Have Security Testing?](#why-have-security-testing)
+    * [How to Get Started with Security Testing?](#how-to-get-started-with-security-testing)
+        * [Phase 1: Before Development Begins](#phase-1-before-development-begins)
+        * [Phase 2: During Definition and Design](#phase-2-during-definition-and-design)
+        * [Phase 3: During Development](#phase-3-during-development)‎
+        * [Phase 4: During Deployment‎](#phase-4-during-deployment)
+        * [Phase 5: Maintenance and Operations](#phase-5-maintenance-and-operations)
+    * [Additional Resources‎](#additional-resources)
+</box>
+
+## What Is Security Testing?
+
+Security Testing is **a type of Software Testing** that uncovers vulnerabilities, threats, risks in a software application and prevents malicious attacks from intruders. The purpose of Security Tests is to identify all possible loopholes and weaknesses of the software system which might result in a loss of information, revenue, repute at the hands of the employees or outsiders of the Organization.
```

Comment:
1. not sure bolding `a type of Software Testing` is necessary
2. identify all possible loopholes -> identify possible loopholes
3. Organization -> organization